node/src/Transaction.ts

```diff
@@ -2182,6 +2183,37 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
         return this.addAndReturn(createZMPop(keys, modifier, count));
     }
 
+    /**
+     * Pops a member-score pair from the first non-empty sorted set, with the given `keys` being
+     * checked in the order they are provided. Blocks the connection when there are no members
+     * to pop from any of the given sorted sets. `BZMPOP` is the blocking variant of `ZMPOP`.
+     * `BZMPOP` is a client blocking command, see {@link https://github.com/valkey-io/valkey-glide/wiki/General-Concepts#blocking-commands | the wiki}
+     * for more details and best practices.
+     *
+     * See https://valkey.io/commands/bzmpop/ for more details.
+     *
```

Comment:
Can we use the "remarks" tag here as well like so, and have a space between:
1. the initial description and the doc link
2. the doc link and the "remarks" tag

We don't seem to have a space between the "remarks" tag and the first param tag, so lets do that here too
```suggestion
     *
     * See https://valkey.io/commands/bzmpop/ for more details.
     *
     * @remarks `BZMPOP` is a client blocking command, see {@link https://github.com/valkey-io/valkey-glide/wiki/General-Concepts#blocking-commands | the wiki}
     * for more details and best practices.
```